docs/learn/learn-proxies.md

```diff
@@ -127,151 +187,137 @@ the chain, as you will be unable to sign extrinsics using the UI.
-window like the one shown below. In this pop-up window, you have an option to clear individual proxy accounts
-or all of them. Under the hood, the UI is calling the extrinsics `removeProxy` for individual accounts and
- `removeProxies` for clearing all of the proxy accounts.
+![nested proxy calls 3x](../assets/nested_proxy_calls_3x.png)
 
-![Remove Proxies](../assets/remove_proxies.png)
+## Anonymous Proxies
 
-The procedure for removing an Anonymous Proxy is different and there are a few functions on the extrinsic page 
-that will help do this.
+:::info
 
-:::warning There is no way to get access to the primary account after deleting the anonymous proxy
+To create an **anonymous proxy account** see [this support article](https://support.polkadot.network/support/solutions/articles/65000182196), or watch [this technical explainer video](https://www.youtube.com/watch?v=T443RcCYP24).
 
-`removeProxy` or `removeProxies` do not work for anonymous proxies. You must use the 
-`killAnonymous` function which must be called **from** the *anonymous* proxy. 
-This means that the anonymous proxy must be added as an account to the Polkadot-JS UI.
+:::
+
+:::note Explainer video on anonymous proxies
+
+Learn more about anonymous proxies from our
+[technical explainer video](https://www.youtube.com/watch?v=YkYApbhU3i0).
 
 :::
 
-The following steps can be used to remove your anonymous proxy:
+:::danger Risk of loss of funds
 
-- **Step 0**: You need to know the following information:
+Read carefully the text below and before performing any action using anonymous proxies on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, experiment on the Westend testnet.
 
-  - the **account** you created the anonymous proxy from
-  - **type of proxy**, index (almost always 0)
-  - **block height** it was created at
-  - the **extrinsic index** in the block (on most block explorers, you will see the extrinsic ID listed as something along the lines of "9000-2" -> 9000 is the block height (block number) and 2 is the extrinsic index. You can find this information by looking up your account in a block explorer.
+:::
 
-  ![anon proxy info](../assets/kill-proxy-1.png)
+Anonymous proxies are very different from other proxy types. Proxies that we described so far are _existing accounts_ assigned as proxies by a primary account. These proxies act on behalf of the primary account reducing the exposure of the primary account's private key. Remember, the more often we use an account's private key to sign for different things, the more we expose that key to the internet, increasing the visibility of that account. The purpose of a proxy is thus to draw the attention of potential attackers away from the primary account, as proxies' private keys will be used most of the time to perform actions on behalf of the primary account. We can imagine proxies as bodyguards of a VIP, loyal and ready to risk their lives to ensure the VIP's protection.
```

Comment:
The proxies <-> bodyguards analogy works for the context of security. In general, the analogy of an organization with different departments, Prime minister/President with different ministers for different tasks and privileges is stated.

```suggestion
Anonymous proxies are very different from other proxy types. The proxies we described so far are _existing accounts_ assigned as proxies by a primary account. These proxies act on behalf of the primary account, reducing the exposure of the primary account's private key. Remember, the more often we use an account's private key to sign transactions, the more we expose that key to the internet, increasing the visibility of that account. The purpose of a proxy is thus to draw the attention of potential attackers away from the primary account, as proxies' private keys will be used most of the time to perform actions on behalf of the primary account. We can imagine proxies as bodyguards of a VIP, loyal and ready to risk their lives to ensure the VIP's protection.
```

Reply:
I will more context in the last commits, beyond security